javaparser-core/src/main/java/com/github/javaparser/utils/Utils.java

```diff
@@ -50,6 +50,13 @@
         return o;
     }
 
+    public static String assertNonEmpty(String id) {
+        if (id == null || id.isEmpty()) {
```

Comment:
That and the error message - dunno, I guess "String s" and "a string was unexpectedly empty" or "This string may not be empty."